hamilton/graph.py

```diff
@@ -169,7 +170,7 @@ def create_function_graph(
         for n in fm_base.resolve_nodes(f, config):
             if n.name in config:
                 continue  # This makes sure we overwrite things if they're in the config...
-            if n.name in nodes:
+            if n.name in nodes and not allow_module_overrides:
                 raise ValueError(
                     f"Cannot define function {n.name} more than once."
                     f" Already defined by function {f}"
```

Comment:
You can add an error message here that links to the docs in case people want to use it -- now that we have a good approach to fixing this. Just make sure to outline the caveats.

Reply:
We can add this after the fact, but on second thought we don't need to advertise this as the first solution as it's a slight power-user case.